source/support.tex

```diff
@@ -1436,11 +1435,15 @@
 
 \begin{itemdescr}
 \pnum
-\tcode{true} if and only if the type adheres to ISO/IEC/IEEE
-60559.
+\tcode{true} if and only if the type adheres to ISO/IEC/IEEE 60559.
 \begin{footnote}
-ISO/IEC/IEEE 60559:2011 is the same as IEEE 754-2008.
+ISO/IEC/IEEE 60559:2020 is the same as IEEE 754-2019.
 \end{footnote}
+\begin{note}
+The value is \tcode{true} for the types
+\tcode{float16_t}, \tcode{float32_t}, \tcode{float64_t}, or \tcode{float128_t},
+if present\iref{basic.extended.fp}.
```

Comment:
That's awkward English.  A phrase with "or" is usually considered to be single, but "the types" is plural.  (That's copied from P1467, so that's my fault for writing that.)

I suggest changing "for the types" to "for any of the types", or changing "or" to "and".  Either of those changes makes the English sound better.  I like the first one ("for any of the types") better.  It should be fine to make this change editorially since it is in a note and doesn't change the meaning of the sentence.

Reply:
Thanks, good call!